ecs-cli/modules/compose/cli/ecs/app/service_command.go

```diff
@@ -136,3 +136,35 @@ func deploymentConfigFlags(specifyDefaults bool) []cli.Flag {
 		},
 	}
 }
+
+func loadBalancerFlags() []cli.Flag {
+	targetGroupArnUsageString := "[Optional] The full Amazon Resource Name (ARN) of the Elastic Load Balancing target group associated with a service."
```

Comment:
Would you mind changing this to:

"[Optional] Specifies the full Amazon Resource Name (ARN) of a previously configured Elastic Load Balancing target group to associate with your service."